etc/systemd/system/legacy.duffy.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+After=network.target
+Requires=primary.duffy.service
+
+[Service]
+User=duffy
+Group=duffy
+ExecStart=/venv/bin/duffy -c /etc/duffy-example-config.yaml serve-legacy
+Restart=on-failure
+RestartSec=60
```

Comment:
Not sure about automatic restarts… The service Duffy depends on (postgresql, redis, httpd) don't have it, so why should we? Mind that any exceptions in FastAPI controller functions or Celery task functions won't cause the app or worker to quit.

Reply:
I have it here to ensure that the service does not end up in a restart loop and then, in a permanent failed state for continual failures. The restart policy ensure that it would wait for at least 60 seconds before retrying a restart. Other services on which ours depend on would run as they do and on restart, they will simply connect as if nothing happened.

Reply:
Yeah, but how would it end up in a restart loop in the first place? Crashes in the controller or task functions are caught by FastAPI/uvicorn or Celery. Anyway, we're not defining the service type, so it's `simple` and `systemctl start ...` won't report any failure. Meh… :shrug: :wink: